test/e2e/fixtures/e2e_suite.go

```diff
@@ -71,8 +71,6 @@ func (s *E2ESuite) BeforeTest(_, _ string) {
 	s.Diagnostics = &Diagnostics{}
 
 	s.DeleteResources(Label)
-	// create database collection
-	s.Persistence.DeleteEverything()
```

Comment:
I've removed this because it was often deleting offloaded data in non-e2e test. Now we delete specifically the e2e workflows.